src/mpyl/project.py

```diff
@@ -308,8 +308,18 @@ def from_config(values: dict):
         )
 
 
+@dataclass(frozen=True)
+class Rancher:
+    project_id: str
```

Comment:
Okay, one last comment about this I promise :D 
Why is this mandatory? From an mpyl perspective I mean

Reply:
It's not really, only if you have `kubernetes.deployment` and it defaults to an empty string.

Reply:
Shouldn't we still be able to provide a default project ID per env though? I mean, we used it like that before and now we want to be able to set the projectId per project, but setting the project IDs per environment also makes sense (even though we want to do something else now)

Reply:
My question was whether we _need_ to set the project ID for every project if it uses Kubernetes Deploy. From an mpyl's perspective, I don't think the project ID should be mandatory per project